README.md

```diff
@@ -1,46 +1,20 @@
+- Inside this folder run `yarn` and then `yarn start` and `yarn test`, [help setting up yarn](https://classic.yarnpkg.com/en/docs/install).
+- CRA requires `Node 14.0.0 or later version`, instructions to [change your node version](https://stackoverflow.com/questions/53785383/how-to-change-node-version-with-nvm/53785482).
+- Create a new private repo on Github and follow the instructions to [push your local code to your remote repository](https://docs.github.com/en/get-started/using-git/pushing-commits-to-a-remote-repository).
+- Check out a new branch locally to complete the challenge
+- When you are ready, push that branch and open a PR
+- Add `funador` as a collaborator on your private repo
+- Any questions or ready for a review? Please send an email to `jesse@extend.com`
+- If you prefer JavaScript, use the [JavaScript branch](https://github.com/helloextend/frontend-challenge-scaffold/tree/typescript) as your starter.
 
-### `yarn start`
+## Challenge
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-### `yarn test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `yarn build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `yarn eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+- Follow the instructions in App.tsx
+- Use any packages that you want
+- Style the UI however you want
```

Comment:
could we maybe have it be with where they only use 1 option though. Like don't mix styled components and `scss` modules or something like that? Or maybe we only ever saw the mixed styles due to us including styled components. I feel like for this actually, we want to see that they know css, so maybe ui-kits? Like they need to write some css.

Reply:
Oh also, maybe we specify to not use class based components and for them to make use of hooks. I know we had some candidates in the past that weren't 100% up to speed on recent versions of React. So maybe we specify that here?

Reply:
I was kind of thinking this open ended and that gives us follow up material in the interview.  I think a good engineer would stick to one css approach, so if there is more than one that would be a flag. 

But why don't we see what happens for the first few and then make adjustments as required?